docs/Ecosystem.md

```diff
@@ -25,6 +26,10 @@ See the [translation](./Translation.md#available-locales) page.
 
 * **[AWS Cognito](https://docs.aws.amazon.com/cognito/latest/developerguide/setting-up-the-javascript-sdk.html)**: [thedistance/ra-cognito](https://github.com/thedistance/ra-cognito)
 
+## Authorization Management
+
+* **[Access Control List for Resources](https://github.com/marmelab/ra-auth-acl)**: [marmelab/ra-auth-provider](https://github.com/marmelab/ra-auth-acl)
```

Comment:
Add the "ACL" acronym in the description - that's what people will type in a search engine.